Dockerfile_krb5

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6-slim
```

Comment:
Let's have a directory called `krb5` and place the Dockerfile inside. Afterwards, we need to make `reana-dev` aware of this and build the image accordingly.